community/kernel/src/main/java/org/neo4j/kernel/api/cursor/PropertyItem.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.api.cursor;
+
+import java.nio.channels.WritableByteChannel;
+
+/**
+ * Property item returned from property cursors
+ */
+public interface PropertyItem
```

Comment:
Javadoc refers to `Cursor<PropertyItem>` right?
`property cursors` sound like there exist something like `PropertyCursor` which is removed.